src/components/Footer.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    Flex,
+    Text,
+    Image,
+    Box,
+    useColorModeValue,
+  } from "@chakra-ui/react";
+
+  import { useLoginStore, LoginStore } from "../stores/login";
+  import { gql, useMutation } from "@apollo/client";
+  
+  const LOGOUT = gql`
+    mutation Logout {
+      logout {
+        success
+      }
+    }
+  `;
+  
+  export const Footer = () => {
+    const loginState = useLoginStore();
+    const bg = useColorModeValue("gray.100", "gray.700");
+  
+    return (
+      <Flex
+        w="100vw"
+        h="20px"
+        position="fixed"
+        bottom= "0px"
+        bg={bg}
+        alignItems="center"
+        justifyContent="space-between"
+        px={3}
```

Comment:
The box also isn't tall enough for the logo it seems:
![image](https://user-images.githubusercontent.com/32287958/174663838-9f0e1013-ec0f-4d28-895a-a690409a3dc6.png)

Reply:
Yeah sounds good I added the margin right so the scroll bar doesnt cut off the link.

I made the box half the height of the logo because i think it adds depth, but I can change if people dont like the way it looks.